extension-points/extension-points-by-plugin.dita

```diff
@@ -9,6 +9,13 @@
   </titlealts>
   <shortdesc>The default plug-ins that ship with DITA Open Toolkit include a series of extension points that can be used
     to modify various aspects of toolkit processing.</shortdesc>
+  <prolog>
+    <metadata>
+      <keywords>
+        <indexterm>plug-ins<indexterm>extension point</indexterm></indexterm>
```

Comment:
This sounds like there's only a single extension point for plug-ins. Use plural for subentry,